extensions/kudo/client/src/main/resources/META-INF/services/io.fabric8.kubernetes.client.ExtensionAdapter

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2018 Red Hat inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+io.fabric8.knative.client.KnativeExtensionAdapter
+io.fabric8.knative.client.serving.v1beta1.ServingV1Beta1ExtensionAdapter
```

Comment:
I don't think you have added `ExtensionAdapter` classes. I think you would need to add `KudoExtensionAdapter` class so that `Kudo` client could be adapted from KubernetesClient. Please have a look at KnativeExtensionAdapter :
https://github.com/fabric8io/kubernetes-client/blob/master/extensions/knative/client/src/main/java/io/fabric8/knative/client/KnativeExtensionAdapter.java